### DISCUSSION TITLE
Has anyone used the "allow_weak_tls_server" option in server.config file

### DISCUSSION THREAD

**Discussion by dgilmore82** [Velocidex/velociraptor#3048] [Q&A] [answered]
Hello,

 I saw this [allow_weak_tls_server](https://docs.velociraptor.app/docs/deployment/references/#:~:text=allow_weak_tls_server) option and wanted to enabled it but I keep getting the following error when I try to enable it under the client -> crypto section:
 
 _**field allow_weak_tls_server not found in type proto.ClientConfig**_
 
 Wanted to see if anyone could point me in the right direction to get this configured.

Thanks!!

**Comment by scudette** [comment]
It needs to be in the crypto section which is in the client section.

```yaml
Client:
  Crypto:
     allow_weak_tls_server: true
```

See the example config file here

https://github.com/Velocidex/velociraptor/blob/master/docs/references/server.config.yaml#L94

**Comment by dgilmore82** [comment]
I'm still getting the following error:
![image](https://github.com/Velocidex/velociraptor/assets/104925486/8701860f-66ff-4d63-b36e-4620cb7c02cb)

Here is how I have it in the server config file:
![image](https://github.com/Velocidex/velociraptor/assets/104925486/8c1192e7-e5e3-4663-86bc-da99920a5409)

I am using version 6.9.0 at the moment as well. Should I update the version or is that option available in the 6.9.0 version?

**Comment by scudette** [comment, marked as answer]
Yes if you click the link above and hit the blame button you will see this option was introduced on 30 Aug 
![image](https://github.com/Velocidex/velociraptor/assets/3856546/5a423f1e-54b1-4a09-9e2b-5a7ce1fb1a87)

While 0.6.9 was release May 25

**Comment by dgilmore82** [comment]
It working as expected now. Thanks!!
